src/main/java/io/vertx/core/VertxOptions.java

```diff
@@ -37,6 +37,11 @@
    */
   public static final int DEFAULT_EVENT_LOOP_POOL_SIZE = 2 * CpuCoreSensor.availableProcessors();
 
+  /**
+   * The default capacity for the queue into the worker pool executor service
+   */
+  public static final int DEFAULT_WORKER_POOL_QUEUE_CAPACITY = Integer.MAX_VALUE;
```

Comment:
it should be unbounded by default as this is the actual behavior.

Reply:
Responded in the comments section -- this was the previous implementation default.